packages/teams-js/src/private/interfaces.ts

```diff
@@ -132,6 +132,15 @@ export interface FilePreviewParameters {
    */
   type: string;
 
+  /**
+   * @hidden
+   * The size of the file in bytes.
+   *
+   * @internal
+   * Limited to Microsoft-internal use
+   */
+  size?: string;
```

Comment:
>size [](http://example.com/codeflow?start=2&length=4)

Can you name this so that the unit is in the name, e.g. `sizeOfFileInBytes`

Reply:
Yes even I was thinking to have the unit in name. I can change it to `sizeInBytes` because I think file is implicit here as with other params.